go/libkb/secret_store_darwin.go

```diff
@@ -222,9 +222,8 @@ func (k KeychainSecretStore) clearSecret(mctx MetaContext, account keychainSlott
 }
 
 func NewSecretStoreAll(mctx MetaContext) SecretStoreAll {
-	if mctx.G().Env.DarwinForceSecretStoreFile() {
-		// Allow use of file secret store for development/testing
-		// on MacOS.
+	if mctx.G().Env.ForceSecretStoreFile() {
+		// Allow use of file secret store for development/testing on MacOS.
```

Comment:
Also mention linux if people don't want to pollute their keyring.

Reply:
I'm going to implement the same thing in my PR for forcing file based with the same variable btw

Reply:
but that would be in different file, like `secret_store_linux.go`, right? darwin is exclusively macos/ios

Reply:
Yeah

Reply:
oh I didn't see filename, then dw about this